ethereumkit/src/main/java/org/p2p/ethereumkit/external/model/ERC20Tokens.kt

```diff
@@ -50,43 +34,67 @@ enum class ERC20Tokens(
         mintAddress = "Dn4noZ5jgGfkntzcQSUZ8czkreiZ1ForXYoV2H8Dm7S1",
         coingeckoId = USDT_COINGECKO_ID,
         tokenIconUrl = "https://assets.coingecko.com/coins/images/325/large/Tether.png?1668148663",
+        replaceTokenName = "Tether USD",
         replaceTokenSymbol = USDT_SYMBOL
     ),
-    UST(
-        contractAddress = "0xa693b19d2931d498c5b318df961919bb4aee87a5",
-        mintAddress = "9vMJfxuKxXBoEa7rM12mYLMwTacLMLDJqHozw96WQL8i",
-        coingeckoId = "terrausd",
-        tokenIconUrl = null // TODO!
+    ETH(
+        contractAddress = "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
+        mintAddress = "7vfCXTUXx5WJV5JADk17DUJ4ksgau7utNKj4b963voxs",
+        coingeckoId = ETH_COINGECKO_ID,
+        tokenIconUrl = "https://assets.coingecko.com/coins/images/279/large/ethereum.png?1595348880",
+        replaceTokenName = ETH_NAME,
+        replaceTokenSymbol = ETH_SYMBOL
     ),
-    CRV(
-        contractAddress = "0xd533a949740bb3306d119cc777fa900ba034cd52",
-        mintAddress = "7gjNiPun3AzEazTZoFEjZgcBMeuaXdpjHq2raZTmTrfs",
-        coingeckoId = "curve-dao-token",
-        tokenIconUrl = "https://assets.coingecko.com/coins/images/12124/large/Curve.png?1597369484"
+    SOL(
+        contractAddress = "0xD31a59c85aE9D8edEFeC411D448f90841571b89c",
+        mintAddress = WRAPPED_SOL_MINT,
+        coingeckoId = SOL_COINGECKO_ID,
+        tokenIconUrl = "https://assets.coingecko.com/coins/images/4128/large/solana.png?1640133422",
+        replaceTokenName = SOL_NAME,
+        replaceTokenSymbol = SOL_SYMBOL
     ),
-    LUNA(
-        contractAddress = "0xbd31ea8212119f94a611fa969881cba3ea06fa3d",
-        mintAddress = "F6v4wfAdJB8D8p77bMXZgYt8TDKsYxLYxH5AFhUkYx9W",
-        coingeckoId = "terra-luna",
-        tokenIconUrl = null // TODO!
+    AVAX(
+        contractAddress = "0x85f138bfee4ef8e540890cfb48f620571d67eda3",
+        mintAddress = "KgV1GvrHQmRBY8sHQQeUKwTm2r2h8t4C8qt12Cw1HVE",
+        coingeckoId = "avalanche-2",
+        tokenIconUrl = "https://assets.coingecko.com/coins/images/12559/large/Avalanche_Circle_RedWhite_Trans.png?1670992574",
+        replaceTokenName = "Avalanche",
```

Comment:
no token symbol?

Reply:
it means that we would not replace the real one